cves/CVE-2009-2693.yml

```diff
@@ -43,51 +43,68 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: The vulnerability allows for an attacker to create, modify, or delete files
-  using a specially crafted Web Application Resource (WAR) file. Tomcat fails
-  to check for directory traversal commands in directory paths using ../ and
-  that is what the attacker uses to craft the WAR file.
-  When the WAR file is deployed on the Tomcat server, the directory traversal
-  command can be used to modify files outside of the web root directory. The 
-  files can be created, modified, or deleted. 
-  Secure information from the web server could be moved out of sccope, or a key
-  file for the server could be deleted which could cause a denial of service.
+description:
+  This vulnerability came up in Apache Tomcat 5 and 6 and allowed attackers to create,
+    delete, alter files, view paths and gain access to other parts of the system they should not have permission to
+    by creating special WAR (Web Application Resource) files to be deployed on the Tomcat Server. This occurred because
+    Tomcat did not check these files for directory traversal attempts. If successful the system would give
+    the attackers to any if not all directories accessible to the Tomcat process. Including "../../bin/catalina.bat" as an enty 

```

Comment:
I think you are missing a word here
> "If successful the system would give the attackers (access) to any if not all directories...

Reply:
Ah yes forgot to add the word access. Thanks